src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/BuildShippingMethodFormSubscriber.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\EventSubscriber;
+
+use Sylius\Bundle\ShippingBundle\Form\EventSubscriber\BuildShippingMethodFormSubscriber as BaseBuildShippingMethodFormSubscriber;
+use Sylius\Component\Shipping\Calculator\CalculatorInterface;
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class BuildShippingMethodFormSubscriber extends BaseBuildShippingMethodFormSubscriber
```

Comment:
Is there any particular reason why we have to extend this logic in core instead of only one implementation in `ShippingBundle`